libs/remix-ui/renderer/src/lib/renderer.tsx

```diff
@@ -1,5 +1,7 @@
 import React, { useEffect, useState } from 'react' //eslint-disable-line
+import QueryParams from '../../../../../apps/remix-ide/src/lib/query-params'
 import './renderer.css'
+const registry = require('../../../../../apps/remix-ide/src/global/registry')
```

Comment:
you shouldn't use the registry here.

Reply:
if really need, it's better to pass an instance of a plugin and to `await plugin.call('fileManager', 'file')` in order to get the current file. (btw you already have that)

Reply:
this is getting handled more properly in https://github.com/ethereum/remix-project/pull/1854